lisa/operating_system.py

```diff
@@ -1814,6 +1814,45 @@ def _update_packages(self, packages: Optional[List[str]] = None) -> None:
         if self._no_repo_enabled.search(output):
             raise RepoNotExistException(self._node.os)
 
+    def grow_os_partition_size(self) -> None:
+        from lisa.tools import Mount
+
+        pv_result = self._node.execute("pvscan -s", sudo=True, shell=True).stdout
+        matched = re.compile(r"(?P<disk>.*)(?P<number>[\d]+)", re.M).match(
+            pv_result.splitlines()[0]
+        )
+        if not matched:
+            self._log.debug("LVM is not enabled on the VM. Does not require LV resize.")
+            return
+        root_partition = self._node.tools[Mount].get_partition_info("/")[0]
+        os_version = self._node.os.information.version
+        if os_version >= "9.2.0":
+            sed = self._node.tools[Sed]
+            sed.substitute(
+                regexp="# use_devicesfile = 1",
+                replacement="use_devicesfile = 1",
+                file="/etc/lvm/lvm.conf",
```

Comment:
add comments for this setting

Reply:
Sorry, I forgot this remove this earlier
Deleted now